xbstrap/base.py

```diff
@@ -75,7 +75,11 @@ def installtree(src_root, dest_root):
 		src_path = os.path.join(src_root, name)
 		dest_path = os.path.join(dest_root, name)
 
-		if os.path.isdir(src_path):
+		if os.path.islink(src_path):
```

Comment:
Add a comment that `islink` needs to be done before `isdir` since `isdir` resolves symlinks.